src/main/webapp/WEB-INF/view/chat.jsp

```diff
@@ -216,12 +216,21 @@ String membersOfConversation = (String) request.getAttribute("membersOfConversat
 
   </script>
 
-  <div id="subscribeButton">
-    <button type="button" class="btn btn-primary" data-toggle="button" aria-pressed="false"
-       autocomplete="off">
-      Subscribe
-    </button>
-  </div>
+
+   <button id="subscriptionButton" class="btn btn-primary"  onclick="myFunction()"> subscribe </button>
+
+  <script>
+    document.getElementById("subscriptionButton").onclick = function()
```

Comment:
You don't really need this since you have the onclick handler on line 220 right?